### ISSUE TITLE
Send commands from second broadlink device fails

### ISSUE THREAD

**Issue by samuellazea** [mlfunston/node-red-contrib-broadlink#60]
I have to broadlink devices home and set up. When I try to send commands from the first one it works without problems but when I try to send commands from the second device it does not work and fails with the next error.

10/29/2020, 6:05:07 PMnode: bucatarie
msg : error
"Error: ENOENT: no such file or directory, open '/mnt//jsonIrCode'"
10/29/2020, 6:05:07 PMnode: bucatarie
msg : error
"SyntaxError: Unexpected token u in JSON at position 0"

![Capture9](https://user-images.githubusercontent.com/6168211/97601488-e514a100-1a12-11eb-807b-75089ddeca04.JPG)

The broadlink/newremote/SharedData/jsonDevice  contains this ( I put xxx for confidential data but kept structure intact):
[{"id":1,"mac":"xxxxx","type":10002,"name":"智能遥控","password":xxxxx,"lock":0,"new":0,"longitude":0,"latitude":0,"subDevice":0,"terminalId":1,"publicKey":[-xx,-xx,-55,7,-xx,-xx,-xx,7,xx,84,64,xx,92,-xx,-xx,xxx]}]

In e-Control I have 2 devices:
![Screenshot_20201029-181213_e-Control](https://user-images.githubusercontent.com/6168211/97601090-76374800-1a12-11eb-8577-881bfeb10aaa.jpg)

Maybe it's related with single device being present in configuration.
In nodered when I try to send a command from the first device to some appliance it works but  not if I try to send from the other device.

Is there anything that I do wrong?
 Thanks in advance

**Comment by mlfunston** [comment]
Looks like you may have a path error to the catalog file?
"Error: ENOENT: no such file or directory, open '/mnt**//**jsonIrCode'"

**Comment by samuellazea** [comment]
Yes. That is stated in the log I posted. But how can I fix this?

**Comment by mlfunston** [comment]
You need to put the correct path to the catalog file location (the jsonIrCode and other json files) in the configuration node of that RM unit.
https://github.com/mlfunston/node-red-contrib-broadlink-control#configuring-node-red-nodes

**Comment by samuellazea** [comment]
Thanks lots. I will try it and let you know.
